src-env (mixdbvendors)/_MicroserviceTemplate_.WebApi/Startup.cs

```diff
@@ -0,0 +1,315 @@
+using _MicroserviceTemplate_.Domain.Authorization;
+using _MicroserviceTemplate_.Domain.Jobs;
+using _MicroserviceTemplate_.Domain.MyTables;
+using _MicroserviceTemplate_.EF.Data.Repositories;
+using _MicroserviceTemplate_.WebApi.Authorization;
+using _MicroserviceTemplate_.WebApi.Controllers;
+using _MicroserviceTemplate_.WebApi.Extensions;
+using Asp.Versioning;
+using Asp.Versioning.ApiExplorer;
+using Benraz.Infrastructure.Authorization.Tokens;
+using Benraz.Infrastructure.Common.AccessControl;
+using Benraz.Infrastructure.Common.BackgroundQueue;
+using Benraz.Infrastructure.Common.DataRedundancy;
+using Benraz.Infrastructure.Gateways.BenrazAuthorization.Auth;
+using Benraz.Infrastructure.Web.Authorization;
+using Benraz.Infrastructure.Web.Filters;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Reflection;
+#if SQLSERVER
+using _MicroserviceTemplate_.EF.SqlServer.Services;
+#else
+using _MicroserviceTemplate_.EF.PostgreSql.Services;
+#endif
+
+namespace _MicroserviceTemplate_.WebApi
+{
+    /// <summary>
+    /// Startup.
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        /// Configuration.
+        /// </summary>
+        public IConfiguration Configuration { get; }
+
+        /// <summary>
+        /// Creates startup instance.
+        /// </summary>
+        /// <param name="configuration">Configuration.</param>
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        /// <summary>
+        /// Configures application services.
+        /// </summary>
+        /// <param name="services">Services.</param>
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+            services.AddCors();
+
+            #if SQLSERVER
```

Comment:
@fuadagazade no need 2 function one function and inside only the different lines

Reply:
Changed